src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayCarRentCost = 40;
+  let discount = 0;
```

Comment:
create two const variables for possible discounts instead of reassigning this one and then you can move return inside if statement using specific discount